Pico8Emulator/Log.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Pico8Emulator {
+	public static class Log {
+		public static void Info(object s) {
+			Console.WriteLine(s);
+		}
+		
+		public static void Error(object s) {
```

Comment:
We should also consider creating log files to store errors / warnings and such. It makes it easy to store them so we can keep a history between different runs of the emulator.

Doesn't need to be done right now, just nice to have for later

Reply:
Yep. I will improve the log file, but I will have to make the log saving disabled by default, really want to save my ssd :x

Reply:
yeah, I get it hahaha

What I did in my game to save performance was to have a debug thread that handled file/console writing. I'll create an issue for it and we can solve that later, it's nothing urgent